drivers/eeprom/eeprom_i2c.c

```diff
@@ -50,7 +53,13 @@ static inline void fill_target_address(uint8_t *buffer, const void *addr) {
     }
 }
 
-void eeprom_driver_init(void) { i2c_init(); }
+void eeprom_driver_init(void) {
+    i2c_init();
+#if defined(EXTERNAL_EEPROM_WP_PIN)
+    setPinInputHigh(EXTERNAL_EEPROM_WP_PIN);
+    /* Note: it's expected that the WP pin would have an external pull-up. */
```

Comment:
This note should go in the documentation, not here, IMO. 

And if it is noted here, above the function call would be better. Something like: 
```suggestion
    // Per documentation, this should have an external pull-up.
    setPinInputHigh(EXTERNAL_EEPROM_WP_PIN);
```

Reply:
Removed comment, added a line to the documentation.